### DISCUSSION TITLE
We do not support your current version of Ubuntu

### DISCUSSION THREAD

**Discussion by ioh21** [Security-Onion-Solutions/securityonion#9086] [Unsupported Versions] [answered]
Hi 
I am trying to install security onion on ubuntu server 20.04.1 LTS and I am getting this error message:
"We do not support your current version of Ubuntu"

I have followed the ubuntu installation guide here https://docs.securityonion.net/en/2.3/installation.html
git clone https://github.com/Security-Onion-Solutions/securityonion
cd securityonion

When I run "sudo bash so-setup-network"
securityonion$ 
Getting started...
We do not support your current version of Ubuntu.

is this version of the ubuntu server not supported? is there another scrip i can run to install so-setup-network?

Many thanks.

Idiris

**Comment by dougburks** [comment, marked as answer]
Are you sure you're using Ubuntu 20.04?

What is the output of the following?
```
cat /etc/os-release
```

**Reply by ioh21** [reply]
my apologies, it's not 20.04.
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.1 LTS
Release:	22.04
Codename:	jammy

**Reply by dougburks** [reply]
Security Onion 2.3 does not support Ubuntu 22.04. Please try using Ubuntu 20.04. 

For more information, please see:
https://docs.securityonion.net/en/2.3/getting-started.html
https://docs.securityonion.net/en/2.3/download.html
https://docs.securityonion.net/en/2.3/installation.html
